mesh/mesh.hpp

```diff
@@ -954,8 +1027,18 @@ class Mesh
    void SetCurvature(int order, bool discont = false, int space_dim = -1,
                      int ordering = 1);
 
-   /** Refine all mesh elements. */
-   void UniformRefinement();
+   /// Refine all mesh elements.
+   /** @param[in] ref_algo  Refinement algorithm. Currently used only for pure
```

Comment:
Can we reformat?

Reply:
Feel free to change the formatting - just make sure both the doxygen source and the built documentation look as desired.